cfgov/v1/handlers/blocks/feedback.py

```diff
@@ -1,6 +1,4 @@
 # -*- coding: utf-8 -*-
-from __future__ import unicode_literals
```

Comment:
Will removing this just postpone the need to rewrite this file once we move to Python 3? Is it possible to use bytestrings `b'something'` when we need to refer to non-unicode strings?

Reply:
Great point. I need to keep my py3 hat on.